src/features/common/InputTypes/BootstrapInput.tsx

```diff
@@ -0,0 +1,41 @@
+import { createStyles, InputBase, Theme, withStyles } from '@material-ui/core';
+
+const BootstrapInput = withStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      'label + &': {
+        marginTop: theme.spacing(3),
+      },
+    },
+    input: {
+      borderRadius: 9,
+      position: 'relative',
+      backgroundColor: theme.palette.background.paper,
+      boxShadow: '0px 3px 6px #00000029',
+      fontSize: 14,
+      padding: '10px 26px 10px 12px',
+      transition: theme.transitions.create(['border-color', 'box-shadow']),
+      // Use the system font instead of the default Roboto font.
+      fontFamily: [
+        '-apple-system',
+        'BlinkMacSystemFont',
+        '"Segoe UI"',
+        'Roboto',
+        '"Helvetica Neue"',
+        'Arial',
+        'sans-serif',
+        '"Apple Color Emoji"',
+        '"Segoe UI Emoji"',
+        '"Segoe UI Symbol"',
```

Comment:
these fonts are not our fonts,

Also, I think we had removed Bootstrap some time ago; is there a reason we need to add it again?

Reply:
No this is a custom component, using material UI

Reply:
fonts I'll remove

Reply:
Done